src/main/java/edu/kis/vh/nursery/stack/IntArrayStackInterface.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.vh.nursery.stack;
+
+public interface IntArrayStackInterface {
```

Comment:
"interface" suffix of the attribute name is wrong and does not show the meaning.
"Array" is also unnecessary.